packages/web/src/containers/swap-container/SwapContainer.tsx

```diff
@@ -7,11 +7,28 @@ import { ThemeState } from "@states/index";
 import { useRouter } from "next/router";
 import { useSwapHandler } from "@hooks/swap/use-swap-handler";
 
+import { TokenModel } from "@models/token/token-model";
+const TOKEN_A: TokenModel = {
```

Comment:
It seems like a good idea to modify the definition of `common/values/token-constant.ts`.

Reply:
done